lib/src/se/llbit/chunky/PersistentSettings.java

```diff
@@ -413,5 +413,14 @@ public static void changeSettingsDirectory(File directory) {
     cacheDir = new File(settingsDir, "cache");
     settings.load(settingsFile);
   }
+
+  public static void setOctreeImplementation(int ordinal) {
+    settings.setInt("octreeImplementation", ordinal);
+    save();
+  }
+
+  public static int getOctreeImplementation() {
+    return settings.getInt("octreeImplementation", 1); // Packed Octree hardcoded as an int, not nice
```

Comment:
That's why we should save it as a string in the settings. :upside_down_face:

Reply:
Changed